### PR TITLE
Command line override of config file options.

### PR DESCRIPTION
**Pull request by scottfrazer**

### CODE REVIEW

File: herc/config.py [hunk 1]
```diff
@@ -42,3 +42,16 @@ def get(*args, **kwargs):
     if _config is None:
         raise ConfigNotLoadedException("No config loaded. Did you forget to call config.load_config()?")
     return _config.get(*args, **kwargs)
+
+def put(*args, **kwargs):
+    """Gets a value from the config file."""
```
Comment (helgridly) [inline herc/config.py]: copy paste docstring :)

File: herc/webservice.py [hunk 2]
```diff
@@ -135,43 +135,61 @@ def sleep(self, secs):
...
-    root_logger.addHandler(file_logger)
-    console_handler = logging.StreamHandler()
-    console_handler.setFormatter(log_formatter)
-    console_handler.setLevel(logging.DEBUG)
-    root_logger.addHandler(console_handler)
-
-    if cli.debug:
-        print('Started Herc in DEBUG mode (port {0})'.format(cli.port))
+    if not os.access(config.get('log_file'), os.W_OK):
+        print('WARNING: coult not open log file for writing: ' + config.get('log_file'))
```
Comment (helgridly) [inline herc/webservice.py]: Can this be EAFP'd?

File: herc/config.py (hunk 1 above)
Comment (scottfrazer) [inline herc/config.py]: Ha woops

File: herc/webservice.py (hunk 2 above)
Comment (scottfrazer) [inline herc/webservice.py]: Yeah it can, I'll change

**Pull request merged**